src/Drivers/SymfonyTinkerwellDriver.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+use Tinkerwell\ContextMenu\Label;
+use Tinkerwell\ContextMenu\OpenURL;
+use Tinkerwell\ContextMenu\SetCode;
+use Tinkerwell\ContextMenu\Submenu;
+use Symfony\Component\HttpFoundation\Request;
+
+class SymfonyTinkerwellDriver extends TinkerwellDriver
+{
+    public function canBootstrap($projectPath)
+    {
+        return file_exists($projectPath . '/public/index.php');
+    }
+
+    public function bootstrap($projectPath)
+    {
+        require_once $projectPath . '/config/bootstrap.php';
```

Comment:
I think you can use :  `require_once $projectPath . '/vendor/autoload.php';`

Reply:
I'm using bootstrap as it set's up the ENV for debug/etc that's later used in the `bootstrap` Tinkerwell function.

Reply:
Maybe you can directly set the ENV up in a separate method of the driver. That way, the driver will also work if the the Symfony project doesn't have a config/bootstrap.php file.